src/applications/edu-benefits/complaint-tool/helpers.js

```diff
@@ -24,7 +24,7 @@ export function fetchInstitutions({ institutionQuery, page }) {
 export function transform(formConfig, form) {
   const formData = transformForSubmit(formConfig, form);
   return JSON.stringify({
-    educationBenefitsClaim: {
+    giBillFeedback: {
```

Comment:
It looks like this is likely causing the 400 response, as the backend is expecting:
```
{
  gi_bill_feedback: {
    form: // form as JSON string
  }
}
```